cdap-app-fabric/src/main/java/co/cask/cdap/gateway/handlers/AppFabricHttpHandler.java

```diff
@@ -137,6 +138,8 @@
 
   private final PreferencesStore preferencesStore;
 
+  private final ConsoleSettingsStore consoleSettingsStore;
```

Comment:
Having to call 2 apis is not cool. Esp. because we don't expose v3 yet in any docs, etc. So user's assumption that v2/reset will remove everything